queries.sql

```diff
@@ -89,9 +102,13 @@ SELECT billing_state, COUNT(*), SUM(total) FROM invoices WHERE billing_country =
 SELECT billing_state, COUNT(*), SUM(total), AVG(total) FROM invoices WHERE billing_country = 'USA' AND billing_state = 'CA' GROUP BY billing_state;
 
 -- The count, total, and average of invoice totals, grouped by state, ordered by average invoice total from highest-to-lowest
+SELECT COUNT(*), AVG(total), SUM(total) FROM invoices GROUP BY billing_state ORDER BY AVG(total) DESC;
```

Comment:
It's unlikely you'd produce a report like this without having the `billing_state` column in the final result set.